implementations/rust/ockam/ockam_command/tests/bats/examples/databases.bats

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# ===== SETUP
+
+setup() {
+  load ../load/base.bash
+  load ../load/orchestrator.bash
+  load ./setup.bash
+  load_bats_ext
+  setup_home_dir
+  skip_if_orchestrator_tests_not_enabled
+  copy_enrolled_home_dir
+}
+
+teardown() {
+  if [[ -z $BATS_TEST_COMPLETED ]]; then
+    echo "Test failed $(pwd)" >&3
+    cat log.txt >&3 || true
+  fi
```

Comment:
Isn't this similar to what we do at `teardown_home_dir()`? Should we modify that function instead?

Reply:
I don't think that's needed, as we only create the log.txt file in the examples test.